andela/caret-bn-frontend
# FT: Supplier should delete accommodation

#### What does this PR do?
- A registered supplier should delete an accommodation

#### Description of Task to be completed?
- Allow a pre-screened supplier to delete accommodations

#### How should this be manually tested?
- Clone this repo
- run npm install
- run npm run dev
- Log in as a `supplier`
- Navigate to a supplier accommodation
- Click the `Delete` Button
- Confirm deletion


#### Any background context you want to provide?
N / A
#### What are the relevant pivotal tracker stories?
#####  #170308267

#### Screenshots (if appropriate)
![Screen Shot 2020-01-09 at 16 48 00](https://user-images.githubusercontent.com/13886438/72077839-9faaaa00-3300-11ea-8c63-fc715c6720fa.png)
![Screen Shot 2020-01-09 at 16 48 14](https://user-images.githubusercontent.com/13886438/72077842-a0434080-3300-11ea-8218-b52ab4fe7551.png)

#### Questions:
